pkg/types/types.go

```diff
@@ -41,7 +41,10 @@ type NetConf struct {
 	// These parameters are exclusive in one config file:
 	//  - Delegates (directly add delegate CNI config into multus CNI config)
 	//  - ClusterNetwork+DefaultNetworks  (add CNI config through CRD, directory or file)
-	Delegates       []*DelegateNetConf  `json:"-"`
+	Delegates []*DelegateNetConf `json:"-"`
+	// Allow delegate DEL message to next when some missing error
+	AllowTryDeleteOnErr bool `json:"allowTryDeleteOnErr"`
```

Comment:
Could you please move it to later, after MultusNamespace because we put optional variable there.

Reply:
done